packages/smarthr-ui/src/components/RadioButton/RadioButton.tsx

```diff
@@ -58,21 +50,16 @@ export const RadioButton = forwardRef<HTMLInputElement, Props>(
   ({ onChange, children, className, required, ...props }, ref) => {
     const { wrapperStyle, innerWrapperStyle, boxStyle, inputStyle, labelStyle } = useMemo(() => {
       const { wrapper, innerWrapper, box, input, label } = radioButton()
+      const actualDisabledAttrs = { disabled: !!props.disabled }
```

Comment:
Boolean化せずとも良い気がしました

```suggestion
      const actualDisabledAttrs = { disabled: props.disabled }
```

Reply:
@Qs-F 
tailwind variantsの挙動に詳しくないのですが、下記の箇所でbooleanとしてtrue, falseを期待しているっぽいです。
undefinedを渡しても動作するかも？ですが、今後の挙動変更の可能性とかも考慮するとbooleanにしておくほうが良いかも？

https://github.com/kufu/smarthr-ui/blob/a080c4df158c8eac9a612e142bc0a5ba315a1814/packages/smarthr-ui/src/components/RadioButton/RadioButton.tsx#L36-L45

Reply:
あっ すみません、defaultVariantsがあると思ってたんですが勘違いだったのでBooleanで良いと思いました！！